<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor</class>
 <widget class="QWidget" name="Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="2" column="0">
    <widget class="QWidget" name="notConnectedWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="notConnectedLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="table">
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>9</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Process Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Thread Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prio</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CPU (%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mem Usage (%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Task Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Delta Task Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Memory (stack/heap)</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause Process</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="text">
        <string>Continue Process</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="termButton">
       <property name="text">
        <string>Term Process</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="killButton">
       <property name="text">
        <string>Kill Process</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>table</tabstop>
  <tabstop>pauseButton</tabstop>
  <tabstop>continueButton</tabstop>
  <tabstop>killButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
